tasks/main.yml

```diff
@@ -157,37 +165,98 @@
     mode=0755
   become_user: "{{ project_user }}"
 
+- name: configure celery and celery beat
+  template: src=celery.conf
+            dest=/etc/supervisor/conf.d/{{ project_name }}-celery-{{ item.name }}.conf
+            owner=root
+            group=root
+            mode=0600
+  with_items:
+    - { name: "default", command: "worker", flags: "--loglevel=INFO" }
+    - { name: "beat", command: "beat", flags: "--schedule={{ root_dir }}/celerybeat-schedule.db --pidfile={{ root_dir }}/celerybeat.pid --loglevel=INFO" }
+  notify:
+    - restart celery
+    - restart celery-beat
+
+- name: ensure celery is running
+  supervisorctl: name={{ project_name }}-celery-default state=started
```

Comment:
This is failing: FAILED! => {"changed": false, "failed": true, "msg": "ERROR (no such process)", "name": "our_project-celery-default"}

The conf file on the server does have `[program:our_project-celery-default]`. I'm guessing that we need to make supervisor reload its config files before trying this, because at this point it doesn't know we've added this process yet.  Or maybe we should do this in a handler or something like that?